wallet/how-to/use-3rd-party-integrations/unity-infura.md

```diff
@@ -0,0 +1,38 @@
+---
+description: Set up MetaMask SDK with Infura in your Unity game.
+sidebar_position: 3
+---
+
+# Set up Infura in Unity
+
+You can set up [MetaMask SDK](../../concepts/sdk/index.md) with [Infura](https://docs.infura.io/) in
+your Unity game.
+
+## Prerequisites
+
+- An Infura API key.
```

Comment:
Seems like a requirement since the user isn't instructed to complete a procedure described in the documentation.